### ISSUE TITLE
NullReferenceException calling NR5CompatibiltyExtensions.GetFullName

### ISSUE THREAD

**Issue by mrward** [mono/monodevelop#6281]
VSMac: 7.7.0.1578
Git revision: 1e5b07f83e5bc2d00a9f5636c4fdfb4f0e6451f6
Build date: 2018-10-09 20:13:47+00
Build branch: release-7.7
Xamarin extensions: b361f54d31bf96c9dd889570fac541245b13ffeb

I thought this might be related to #6232 but the callstack seems to be different.

```
Exception in tooltip provider MonoDevelop.SourceEditor.Wrappers.TooltipProviderWrapper GetItem:
System.NullReferenceException: Object reference not set to an instance of an object
  at MonoDevelop.Ide.TypeSystem.NR5CompatibiltyExtensions.GetFullName (Microsoft.CodeAnalysis.ITypeSymbol type) [0x00000] in /Users/vsts/agent/2.140.2/work/1/s/monodevelop/main/src/core/MonoDevelop.Ide/MonoDevelop.Ide.TypeSystem/NR5CompatibiltyExtensions.cs:155 
  at MonoDevelop.CSharp.Completion.CSharpCompletionTextEditorExtension.GetInfo (Microsoft.CodeAnalysis.SyntaxNode root, Microsoft.CodeAnalysis.SemanticModel semanticModel, Microsoft.CodeAnalysis.SyntaxNode node, System.String textOpt, System.Threading.CancellationToken cancellationToken) [0x001e9] in /Users/vsts/agent/2.140.2/work/1/s/monodevelop/main/src/addins/CSharpBinding/MonoDevelop.CSharp.Completion/CSharpCompletionTextEditorExtension.IDebuggerExpressionResolver.cs:141 
  at MonoDevelop.CSharp.Completion.CSharpCompletionTextEditorExtension+<MonoDevelop-Debugger-IDebuggerExpressionResolver-ResolveExpressionAsync>d__53.MoveNext () [0x00262] in /Users/vsts/agent/2.140.2/work/1/s/monodevelop/main/src/addins/CSharpBinding/MonoDevelop.CSharp.Completion/CSharpCompletionTextEditorExtension.IDebuggerExpressionResolver.cs:65 
--- End of stack trace from previous location where exception was thrown ---
  at MonoDevelop.SourceEditor.DebugValueTooltipProvider+<GetItem>d__4.MoveNext () [0x00108] in /Users/vsts/agent/2.140.2/work/1/s/monodevelop/main/src/addins/MonoDevelop.SourceEditor2/MonoDevelop.SourceEditor/DebugValueTooltipProvider.cs:103 
--- End of stack trace from previous location where exception was thrown ---
  at MonoDevelop.SourceEditor.Wrappers.TooltipProviderWrapper+<GetItem>d__6.MoveNext () [0x0008b] in /Users/vsts/agent/2.140.2/work/1/s/monodevelop/main/src/addins/MonoDevelop.SourceEditor2/MonoDevelop.SourceEditor.Wrappers/TooltipProviderWrapper.cs:81 
--- End of stack trace from previous location where exception was thrown ---
  at Mono.TextEditor.TextArea+<TooltipTimer>d__386.MoveNext () [0x000dc] in /Users/vsts/agent/2.140.2/work/1/s/monodevelop/main/src/addins/MonoDevelop.SourceEditor2/Mono.TextEditor/Gui/TextArea.cs:3254 

```

> VS bug [#702248](https://devdiv.visualstudio.com/DevDiv/_workitems/edit/702248)

**Comment by mkrueger** [comment]
No is not related.

**Comment by xamarin-release-manager** [comment]
Fixed in version 7.8.0.800 (master)

Pull Request #6291 merged by: Lluis Sanchez
Author: mono
Commit: 5089174e77a0280aad3cd34a83854a3abd822da6 (mono/monodevelop)

**Comment by xamarin-release-manager** [comment]
Fixed in version 7.7.0.1613 (release-7.7)

Pull Request #6292 merged by: Lluis Sanchez
Author: mono
Commit: ce586e235e3bfdc8f8028f14382b6e902b918222 (mono/monodevelop)

**Comment by xamarin-release-manager** [comment]
Fixed in version 7.6.10.10 (release-7.6)

Pull Request #6296 merged by: Lluis Sanchez
Author: mono
Commit: 7a7fc2772a2ea166dad119de3362177c4c8aa0ce (mono/monodevelop)

**Comment by xamarin-release-manager** [comment]
Fixed in version 8.0.0.240 (release-8.0)

Pull Request #6291 merged by: Lluis Sanchez
Author: mono
Commit: 5089174e77a0280aad3cd34a83854a3abd822da6 (mono/monodevelop)

**Comment by xamarin-release-manager** [comment]
Fixed in version 8.6.0.802 (master)

Pull Request #6291 merged by: Lluis Sanchez
Author: mono
Commit: 5089174e77a0280aad3cd34a83854a3abd822da6 (xamarin/monodevelop)
